README.md

```diff
@@ -155,33 +159,45 @@ Update the project:
 ```bash
 npx cap update ios
 ```
+Sync the project:
 
+```bash
+npx cap sync ios
+```
 Make the following changes to your `AppDelegate.swift` file:
+> To consider: Read the official documentation. The import method is different depending on the swift version. Capacitor version 1.4.0 uses Swift version 4.2. 
+[Check version](https://github.com/ionic-team/capacitor/blob/master/.swift-version)
```

Comment:
I don't understand. How does the import method change?

The Branch docs only show an example for Swift 4.2 but not Swift 4.0

Reply:
I don't think it's necessary to document that, you're right. Currently capacitor uses version 4.2, therefore I do not think this contributes anything.
![alt text](https://i.ibb.co/XZKntHx/Captura-de-pantalla-2020-01-23-a-las-8-49-21.png "IOS BranchIO Doc")